.github/workflows/phpunit-l7.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3, 7.2.5]
+                php: [8.0snapshot, 7.4, 7.3, 7.2.5]
```

Comment:
Any reason for using `8.0snapshot` over `8.0`?

Reply:
My bad. I thought you were using travisci which only has snapshot as of now. Will update.